NightscoutServiceKit/OTPManager.swift

```diff
@@ -74,41 +74,49 @@ public class OTPManager {
     let issuerName = "Loop"
     var tokenPeriod: TimeInterval
     var passwordDigitCount = 6
+    let maxOTPsToAccept: Int
+    
     public static var defaultTokenPeriod: TimeInterval = 30
+    public static var defaultMaxOTPsToAccept = 2
     
-    public init(secretStore: OTPSecretStore = KeychainManager(), nowDateSource: @escaping () -> Date = {Date()}, tokenPeriod: TimeInterval = OTPManager.defaultTokenPeriod) {
+    public init(secretStore: OTPSecretStore = KeychainManager(), nowDateSource: @escaping () -> Date = {Date()}, tokenPeriod: TimeInterval = OTPManager.defaultTokenPeriod, maxOTPsToAccept: Int = OTPManager.defaultMaxOTPsToAccept) {
         self.secretStore = secretStore
         self.nowDateSource = nowDateSource
         self.tokenPeriod = tokenPeriod
+        self.maxOTPsToAccept = maxOTPsToAccept
         if secretStore.tokenSecretKey() == nil || secretStore.tokenSecretKeyName() == nil {
             resetSecretKey()
         }
     }
     
```

Comment:
This method will now throw so we can propagate the precise error that occurred and make sure to log it (from client call above).